package.json

```diff
@@ -89,8 +95,10 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,json,css,md}": [
-      "prettier --no-semi --single-quote --write",
+    "src/**/*.{js,jsx}": [
```

Comment:
I'd like to continue to apply prettier to multiple file types (markdown, etc.). `lint-staged` can accept multiple comma-separated tasks, so how about breaking it up? something like:
```
"lint-staged": {
    "*.{js,jsx,json,css,md}": "prettier --no-semi --single-quote --write",
    "src/**/*.{js,jsx}": [
        "yarn lint",
        "yarn lint:style"
    ],
    "*": "git add"
}
```

Reply:
Of course @marcgarreau, this is a good solution.
I have added a commit with this.